### PR TITLE
Add cancer_gene flag to get_gene_information

### PR DESCRIPTION
**Pull request by mathieulemieux**
See https://www.bcgsc.ca/jira/browse/KBDEV-1129

### CODE REVIEW

File: graphkb/genes.py [hunk 1]
```diff
@@ -444,6 +444,10 @@ def get_gene_information(
     gene_flags["oncogene"] = convert_to_rid_set(get_oncokb_oncogenes(graphkb_conn))
     logger.info("fetching tumour supressors list")
     gene_flags["tumourSuppressor"] = convert_to_rid_set(get_oncokb_tumour_supressors(graphkb_conn))
+    logger.info("fetching cancer_gene list")
+    gene_flags["cancer_gene"] = convert_to_rid_set(get_cancer_genes(graphkb_conn))
+    logger.info("fetching cancer_gene list")
+    gene_flags["cancer_gene"] = convert_to_rid_set(get_cancer_genes(graphkb_conn))
```
Comment (dustinbleile) [inline graphkb/genes.py:451]: Hi @mathieulemieux  - it looks like you have the same lines twice?

File: graphkb/genes.py [hunk 2]
```diff
@@ -444,6 +444,10 @@ def get_gene_information(
     gene_flags["oncogene"] = convert_to_rid_set(get_oncokb_oncogenes(graphkb_conn))
     logger.info("fetching tumour supressors list")
     gene_flags["tumourSuppressor"] = convert_to_rid_set(get_oncokb_tumour_supressors(graphkb_conn))
+    logger.info("fetching cancer_gene list")
+    gene_flags["cancer_gene"] = convert_to_rid_set(get_cancer_genes(graphkb_conn))
```
Comment (dustinbleile) [inline graphkb/genes.py:448]: Hi @mathieulemieux  - did you  check that IPR is already able to accept 'cancer_gene'?

Hi @Nithriel, @mathieulemieux 
Is there a reason why 'cancer_gene' is not camel case like all the other gene flags?  eg. (cancerRelated, tumourSuppressor)

Comment (mathieulemieux) [reply graphkb/genes.py:448]: >did you check that IPR is already able to accept 'cancer_gene'?
Work needed to be done on both side, gkb and ipr, and I don't thing it matters which one first, as long as we make sure it's done on both sides before release. @Nithriel should tell us when the work on her side is done.

>Is there a reason why 'cancer_gene' is not camel case like all the other gene flags?
I thought we agreed on that before. I can change it to camel case if not too late from the ipr side. @Nithriel , what is the exact flag you're using on your side?

Comment (mathieulemieux) [reply graphkb/genes.py]: cancer_gene changed for concerGene.

File: tests/test_genes.py [hunk 3]
```diff
@@ -23,7 +23,7 @@
 
 CANONICAL_ONCOGENES = ["kras", "nras", "alk"]
 CANONICAL_TS = ["cdkn2a", "tp53"]
-CANONICAL_CG = ["ercc1", "fanci", "h2bc4", "h2bc17", "acvr1b"]
```
Comment (dustinbleile) [inline tests/test_genes.py:26]: just curious, were these genes not good tests?

**Review by dustinbleile — APPROVED** [review]

**Review by dustinbleile — APPROVED** [review]

File: tests/test_genes.py (hunk 3 above)
Comment (mathieulemieux) [reply tests/test_genes.py:26]: I first chose these gene randomly among the 500+ I had on hand, then I replaced them with gene 'alb' that I knew I had a related Statement in GKB. The list is here to be fill with more 'canonical' ones once we have some specific use cases we want to test for. I didn't wanted to make it looks as these genes were 'special', so I removed them.

**Pull request merged**